apps/mobile/src/components/CurrencySelect/components/CurrenciesList.tsx

```diff
@@ -1,27 +1,18 @@
+import {FlashList} from '@shopify/flash-list'
 import {
   type CurrencyCode,
   type CurrencyInfo,
 } from '@vexl-next/domain/src/general/currency.brand'
-import {type Atom, type WritableAtom} from 'jotai'
+import {type Atom} from 'jotai'
 import React, {useCallback} from 'react'
-import {FlatList} from 'react-native'
 import {Stack} from 'tamagui'
 import atomKeyExtractor from '../../../utils/atomUtils/atomKeyExtractor'
 import CurrencySelectListItem from './CurrencySelectListItem'
 
 interface Props {
   currencies: Array<Atom<CurrencyInfo>>
   selectedCurrencyCodeAtom: Atom<CurrencyCode | undefined>
-  onItemPress: () => void
-  updateCurrencyLimitsAtom: WritableAtom<
-    null,
-    [
```

Comment:
This could be moved to one type that is used by `CurrenciesList` and `CurrencySelectListItem`